lib/portlayer/storage/vsphere/datastore.go

```diff
@@ -301,3 +303,36 @@ func (d *Datastore) mkRootDir(ctx context.Context, rootdir string) error {
 func (d *Datastore) rootDir() string {
 	return strings.SplitN(d.RootURL, " ", 2)[1]
 }
+
+// Parse the datastore format ([datastore1] /path/to/thing) to groups.
+var datastoreFormat = regexp.MustCompile(`\[([\w\d]*)\]\ (.*)?`)
```

Comment:
Is this deliberately allowing `[] /path/to/thing` and `[]`?

`(.*)?` - ? is redundant as .* will greedily consume everything up to the end of the line so it's not possible to have more than one group here.